examples/HelloVertices.cpp

```diff
@@ -80,8 +80,9 @@ void init() {
 void frame() {
     static const uint32_t vertexBufferOffsets[1] = {0};
     nxt::RenderPass renderpass = device.CreateRenderPassBuilder()
-        .EnableAttachment(0, nxt::TextureFormat::R8G8B8A8Unorm)
-        .SetSubpassColorAttachment(0, 0, 0)
+        .SetAttachmentCount(1)
+        .AttachmentSetFormat(0, nxt::TextureFormat::R8G8B8A8Unorm)
+        .SubpassSetColorAttachment(0, 0, 0)
```

Comment:
Currently, subpass count defaults to 1, attachment count defaults to 0. They aren't required.